src/monitor/pgautofailover--1.3--1.4.sql

```diff
@@ -0,0 +1,733 @@
+--
+-- extension update file from 1.3 to 1.4
```

Comment:
I haven't looked at this script in detail, but my simple heuristic of diffing the output of `\dx+ pgautofailover` on a 1.4 node and a 1.3 upgraded to 1.4 node resulted in these aditional lines on the upgraded version:
```
 function pgautofailover.get_other_nodes(text,integer)
 function pgautofailover.set_node_nodename(bigint,text)
 sequence pgautofailover.event_eventid_seq1
 sequence pgautofailover.node_nodeid_seq1
```

NOTE: these sequences (without the suffixed `1`) exist on both, so I guess it is creating an extra one by accident.
```
 sequence pgautofailover.event_eventid_seq
 sequence pgautofailover.node_nodeid_seq
```

Reply:
Fixed now, thanks! That `\dx+` diff test is a good one. I wish we could script it in the test suite, but I don't think it's possible at the moment. About the sequences, that's because I created the new table structure using `bigserial` instead of manually attaching to the pre-existing sequence.